### ISSUE TITLE
Is this a bug in g2c_compare.c?

### ISSUE THREAD

**Issue by DusanJovic-NOAA** [NOAA-EMC/NCEPLIBS-g2c#515]
In this [section](https://github.com/NOAA-EMC/NCEPLIBS-g2c/blob/develop/utils/g2c_compare.c#L71-L74) in g2c_compare.c:

```C
 69     /* Open the two files. */
 70     g2c_set_log_level(10);
 71     if ((ret = g2c_open(path[0], G2C_NOWRITE, &g2cid0)))
 72         return ret;
 73     if ((ret = g2c_open(path[0], G2C_NOWRITE, &g2cid1)))
 74         return ret;
```
shouldn't the second file use `path[1]` instead of `path[0]`?

**Comment by DusanJovic-NOAA** [comment]
```
$ ../NCEPLIBS-g2c/install/bin/g2c_compare
Segmentation fault (core dumped)
```
Running the program with no command line arguments should not segfault, it should print some kind of help/usage info.

**Comment by edwardhartnett** [comment]
Uggg! What a dumb bug.

I've fixed it in #530 

Thanks @DusanJovic-NOAA for catching this!
